homeassistant/components/openhardwaremonitor/sensor.py

```diff
@@ -30,8 +30,8 @@
 CONF_INTERVAL = "interval"
 
 MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=15)
-SCAN_INTERVAL = timedelta(seconds=30)
-RETRY_INTERVAL = timedelta(seconds=30)
+SCAN_INTERVAL = timedelta(seconds=1)
```

Comment:
We don't allow a default scan interval below 5 seconds even for local polling integrations. And as already pointed out, setting the scan interval lower than the minimum time between updates seems useless.

The user can already customize the scan interval in the platform options for this integration.